### ISSUE TITLE
Error: Node type 'ClassPrivateProperty' is not supported yet

### ISSUE THREAD

**Issue by meezwhite** [putoutjs/minify#2]
Trying to minify through `minify index.js > index.min.js` using [coderaiser/minify](https://github.com/coderaiser/minify). I get the following error stack:

```bash
Error: Node type 'ClassPrivateProperty' is not supported yet: '#random;'
    at maybe.maybeThrow (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1337830)
    at f (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1341416)
    at file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1341886
    at classVisitor (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1298308)
    at _class.ClassDeclaration (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1298619)
    at maybe.maybePlugin (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1338215)
    at f (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1341531)
    at file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1341886
    at Array.forEach (<anonymous>)
    at Program (file:///project/node_modules/@putout/minify/bundle/minify.min.js:1:1326883)
```

In my case the error seems to occur at [p5.grain.js#L17](https://github.com/meezwhite/p5.grain/blob/update-minify/p5.grain.js#L17)

To replicate error (macOS-only):
1. Checkout [p5.grain/tree/update-minify](https://github.com/meezwhite/p5.grain/tree/update-minify)
2. `npm i`
3. `npm run build:darwin`

**Comment by coderaiser** [comment]
Fixed in [`@putout/printer@1.103.0`](https://github.com/putoutjs/printer/releases/tag/v1.103.0)   and [`@putout/minify@1.16.5`](https://github.com/putoutjs/minify/releases/tag/v1.16.5)🎉.

Is it works for you?

**Comment by meezwhite** [comment]
`ClassPrivateProperty` works now, thanks! 🎉

However now `Error: Node type 'ClassPrivateMethod' is not supported yet`, see #3.
